query-connector/src/app/database-service.ts

```diff
@@ -829,6 +829,47 @@ export async function getCustomQueries(): Promise<CustomUserQuery[]> {
   return Object.values(formattedData);
 }
 
+/**
+ * Deletes a query from the database by its unique ID.
+ * @param queryId - The unique identifier of the query to delete.
+ * @returns A success or error response indicating the result.
+ */
+export const deleteQueryById = async (queryId: string) => {
+  // TODO: should be able to simplified when it is just deleting query table
+  const deleteQuerySql1 = `
+    DELETE FROM query_included_concepts 
+    WHERE query_by_valueset_id IN (
+      SELECT id FROM query_to_valueset WHERE query_id = $1
+    );
+  `;
+  const deleteQuerySql2 = `
+    DELETE FROM query_to_valueset WHERE query_id = $1;
+  `;
+  const deleteQuerySql3 = `
+    DELETE FROM query WHERE id = $1;
+  `;
+
+  const client = await dbClient.connect();
```

Comment:
is there a reason we're making a new client connection and not just using the one that we instantiate earlier in this file?

Reply:
think we might also be getting to the point that we start splitting up this file since it's getting pretty big

Reply:
I think it was getting a little fussy when I was first working with the multiple deletions, but yeah, I can look into fixing this.

Reply:
agreed on splitting out the file. I can create a tech debt ticket to address.

Reply:
https://linear.app/skylight-cdc/issue/QUE-108/break-up-database-servicests-to-be-more-manageable